designdocs/CAPACITY.md

```diff
@@ -0,0 +1,148 @@
+# Capacity Design Doc
+
+## Context and Scope
+
+In this document I will introduce the concept of Capacity which will provide a way for users to obtain fee-less transactions. 
```

Comment:
This section needs to define the scope of the design document by saying what is and is not specified.